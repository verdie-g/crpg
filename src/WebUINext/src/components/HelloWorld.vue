<script setup lang="ts">
interface Props {
  msg: string;
}

defineProps<Props>();
</script>

<template>
  <div>{{ msg }}</div>
</template>
