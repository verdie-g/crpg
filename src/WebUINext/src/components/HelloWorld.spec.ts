import { mount } from '@vue/test-utils';
import HelloWorld from './HelloWorld.vue';

it('displays message', () => {
  const wrapper = mount(HelloWorld, {
    props: {
      msg: 'Hello world',
    },
  });

  expect(wrapper.text()).toContain('Hello world');

  expect(wrapper.html()).toMatchSnapshot();
});
