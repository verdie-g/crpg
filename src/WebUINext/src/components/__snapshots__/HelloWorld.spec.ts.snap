// Vitest Snapshot v1

exports[`displays message 1`] = `"<div>Hello world</div>"`;
